### ISSUE TITLE
Naming a file with the same name as an imported library causes false alert

### ISSUE THREAD

**Issue by SuhairZain** [bcherny/tslint-no-circular-imports#2]
We have a file named `history.ts` which takes care of initializing the history. It uses the `history` module from NPM within it, which causes the lint to produce false alerts. The code is like so:
```ts
// history.ts

import { createMemoryHistory } from 'history';
```
This causes an error for dependency chain history -> history. Is there a way to easily fix this issue, other than to rename our file?

**Comment by bcherny** [comment]
@SuhairZain Thanks for the report - I just pushed a fix. Try installing from master (`npm install bcherny/tslint-no-circular-imports`) - can you confirm that fixes your issue?

**Comment by kraenhansen** [comment]
@bcherny any change this would get released on npm? I did `npm install tslint-no-circular-imports` and ran into this issue.

**Comment by bcherny** [comment]
@kraenhansen Published as 0.2.1- thanks for the heads-up.
